meson.build

```diff
@@ -23,142 +23,183 @@ project(
 
 lib_version = '0.5.0'
 
+# TODO: pkg-config
+# TODO: cmake-config
+
+inc_include = include_directories('include')
+
+#####################
+# Config Generation #
+#####################
+
 cc = meson.get_compiler('c')
+cc_native = meson.get_compiler('c', native: true)
```

Comment:
`cc` and `cc_native` definitions are rather trivial. I think you could inline them into the `cc_dict` and `cc_native_dict`.